blockchains/smartchain/assets/0x551FAAB1027Cc50eFAeA5BED092E330475C3Cd99/info.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "MonbaseCoin",
+  "type": "BEP20",
+  "symbol": "MBC",
+  "decimals": 18,
+  "website": "https://monbase.com/",
+  "description": "",
```

Comment:
Please add your description

Reply:
"description": "Monbase is a digital asset exchange offering hundreds of cryptocurrencies for buying, selling, and trading. Users can register, complete KYC, deposit funds, and manage their investments using an exchange wallet. Live market data is also provided for market analysis.",